torch/nn/utils/rnn.py

```diff
@@ -136,7 +137,10 @@ def pack_padded_sequence(input, lengths, batch_first=False):
     return PackedSequence(data, batch_sizes)
 
 
-def _symbolic_pad_packed_sequence(g, input, batch_first=False, padding_value=0.0):
+def _symbolic_pad_packed_sequence(g, input, batch_first=False, padding_value=0.0, total_length=None):
+    if total_length is not None:
+        raise ValueError("_symbolic_pad_packed_sequence only supports "
+                         "_symbolic_pad_packed_sequence=None")
```

Comment:
you probably meant `total_length=None` 😄

Reply:
haha yeah, let me update that!